src/models/document/document-content.ts

```diff
@@ -528,8 +528,8 @@ export const DocumentContentModel = types
     }
   }))
   .actions(self => ({
-    addTileContentInNewRow(content: ToolContentModelType, options?: INewTileOptions): INewRowTile {
-      const title = self.getNewTileTitle(content);
+    addTileContentInNewRow(content: ToolContentModelType, options?: INewTileOptions, tileTitle?: string): INewRowTile {
```

Comment:
In the interest of keeping the API simple, I'd suggest putting the `title` as an optional property in `INewTileOptions` rather than having multiple optional parameters, which can lead to minor annoyances like having to do `addTileContentInNewRow(content, undefined, title)`, for instance.